src/Workspaces/Core/Portable/Diagnostics/CodeAnalysisEventSource.Workspaces.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.CodeAnalysis
+{
+    [EventSource(Name = "Microsoft-CodeAnalysis-Workspaces")]
```

Comment:
Are we going to need to ask the RPS folks to turn this collector on?

Reply:
I am expecting the following PR to enable this collector: https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/587560